loads-broker.tpl

```diff
@@ -3,28 +3,28 @@ source molotov.env && \
 echo "Building loads-broker.json" && \
 cat > loads-broker.json <<EOF
 {
-  "name": "Loadtest: Page Shot Server Testing",
+  "name": "Loadtest: Firefox Screenshots Server Testing",
   "plans": [
 
     {
       "name": "Loadtest",
       "description": "4 boxes",
       "steps": [
         {
-          "name": "Page Shot",
+          "name": "Screenshots",
           "instance_count": 4,
           "instance_region": "us-east-1",
           "instance_type": "m3.medium",
           "run_max_time": 3600,
-          "container_name": "firefoxtesteng/pageshot-loadtests:latest",
+          "container_name": "firefoxtesteng/screenshots-loadtests:latest",
           "environment_data": [
-            "URL_SERVER=https://pageshot.stage.mozaws.net",
+            "URL_SERVER=https://screenshots.stage.mozaws.net",
             "TEST_PROCESSES=10",
             "TEST_DURATION=600",
             "TEST_CONNECTIONS=100"
           ],
           "volume_mapping": "/var/log:/var/log/$RUN_ID:rw",
-          "docker_series": "pageshot_tests"
+          "docker_series": "screenshots_tests"
```

Comment:
Wasn't 💯 on what this change was.

https://github.com/loads/loads-broker/blob/master/JSON.rst says:

> `docker_series` (String, optional): The InfluxDB time series containing CPU and memory stats for this step. Defaults to `"stats"`.